packages/nimble-components/src/table/index.ts

```diff
@@ -138,22 +149,31 @@ export class Table<
     private readonly tableValidator = new TableValidator();
     private readonly updateTracker = new UpdateTracker(this);
     private columnNotifiers: Notifier[] = [];
+    private readonly collapsedRows = new Set<string>();
 
     public constructor() {
         super();
         this.options = {
             data: [],
             onStateChange: (_: TanStackUpdater<TanStackTableState>) => {},
+            onExpandedChange: this.handleExpandedChange,
             getCoreRowModel: tanStackGetCoreRowModel(),
             getSortedRowModel: tanStackGetSortedRowModel(),
+            getGroupedRowModel: tanStackGetGroupedRowModel(),
+            getExpandedRowModel: tanStackGetExpandedRowModel(),
+            getIsRowExpanded: this.getIsRowExpanded,
             columns: [],
-            state: {},
+            state: {
+                grouping: [],
+                expanded: true // Workaround until we can apply a fix to TanStack regarding leveraging our getIsRowExpanded implementation
```

Comment:
Is there a TanStack bug or Nimble issue you can link to to track removing this?

Reply:
Not yet, but there will be. The issue with filing an issue to TanStack is that they require a standalone reproducible example of the issue you're trying to resolve. That will take time to put together, and I certainly don't want to gate this PR on that.

Reply:
Created issue #1162.